pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/rule/codestyle/VariableNamingConventionsRule.java

```diff
@@ -44,49 +45,49 @@
     private static final BooleanProperty CHECK_PARAMETERS_DESCRIPTOR = new BooleanProperty("checkParameters",
             "Check constructor and method parameter variables", true, 3.0f);
 
-    private static final StringMultiProperty STATIC_PREFIXES_DESCRIPTOR = new StringMultiProperty("staticPrefix",
-            "Static variable prefixes", new String[] { "" }, 4.0f, ',');
+    private static final PropertyDescriptor<List<String>> STATIC_PREFIXES_DESCRIPTOR = stringListProperty("staticPrefix").desc("Static variable prefixes").defaultValues("").delim(',').build();
```

Comment:
Maybe we should enable the checkstyle rule to enforce not too long lines... it wouldn't be a problem in the IDE, but on github, it's not so convenient to scroll horizontally... :smile: